client-app/ui-kit/components/atoms/alert/vc-alert.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <div class="vc-alert" :class="[type && `vc-alert--${type}`]">
     <svg v-if="icon" class="vc-alert__icon">
-      <use :href="iconSrc"></use>
+      <use :href="iconSrc" />
```

Comment:
Could be fast refactor computed value  iconSrc, because images in switch like"x-circle", "exclamation-circle" and etc. are duplecated images from "/static/icons/basic/..." ? (see comment above )